packages/oauth/src/index.spec.js

```diff
@@ -216,6 +218,43 @@ describe('OAuth', async () => {
               assert.fail(error.message);
           }
       });
+      it('should return a generated url when passed a custom authorizationUrl', async () => {
```

Comment:
does this test also cover the new `clientOptions` option for `InstallProvider`?

Reply:
No :(. I wanted to but I think I need to change my instrumentation around how I rewire `Webclient`

Reply:
seems pretty low risk to me, but can we create an issue (labeled `tests`) to come back to this? it might even be a `good first issue` for someone.